### PR TITLE
Issue/quick 217 recommendations as artefacts

### PR DESCRIPTION
**Pull request by kema2000**

### CODE REVIEW

**Review by mminns — APPROVED** [review]

**Comment by dagguh** [comment]
Let's rebase to fix conflicts and let's get rid of that merge commit

File: src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareExploration.kt [hunk 1]
```diff
@@ -70,7 +71,11 @@ class HardwareExploration(
         val awsExecutor = Executors.newFixedThreadPool(awsParallelism)
         val explorationExecutor = Executors.newFixedThreadPool(space.size)
         try {
-            return exploreHardwareInParallel(space, explorationExecutor, awsExecutor)
+            val result = exploreHardwareInParallel(space, explorationExecutor, awsExecutor)
+            if (failedHardware.isNotEmpty()) {
+                throw Exception("One or more tests failed : $failedHardware")
```
Comment (dagguh) [inline src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareExploration.kt]: Let's rebase to fix conflicts and let's get rid of any merge commits

**Review by dagguh — CHANGES_REQUESTED** [review]

File: src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareExploration.kt (hunk 1 above)
Comment (dagguh) [reply src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareExploration.kt]: Ok, I did it and force-pushed it here

**Review by dagguh — APPROVED** [review]

**Comment by kema2000** [comment]
Thanks @dagguh

**Pull request merged**